frontend/src/containers/ProductDetails2.jsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import ImgProductCamera from "../assets/img/product-camera.jpg";
+import { connect } from 'react-redux';
+
```

Comment:
I would like you to refer to this project to work on the detail screen. This is Twitter App created in React & Redux + Django. I added it to your GitHub to read this repository.
https://github.com/Tech-i-s/techis-wd-twitter-django-react

Please check out this page. As you can understand, we are not using the `componentWillMount` and `mapStateToProps`. Instead, we use `useEffect` and `useSelector`.

`frontend/src/templates/Edit.jsx`
https://github.com/Tech-i-s/techis-wd-twitter-django-react/blob/main/frontend/src/templates/Edit.jsx

Reply:
Changed to functional component. This is done.